release-notes.md

```diff
@@ -8,6 +8,43 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
+## January 13, 2020
+
+The following updates were made in this release:
+
+**iOS Griffon 1.0.3**
+
+* Griffon bridge and Griffon SDK unified. Setup steps are slightly different now. See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/set-up-project-griffon) for updated documentation.
+* Adobe Analytics debug flag now enabled on Griffon session start and disabled on session stop.
+* Client-side UI now logs location service events (entry/exit) events.
+* Updated for iOS 13 compatibility.
+* Fixed a crash of LIBDISPATCH.
+
+**Android Griffon 1.1.0**
+* Griffon bridge and Griffon SDK unified. Setup steps are slightly different now. See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/set-up-project-griffon) for updated documentation.
+* Adobe Analytics debug flag now enabled on Griffon session start and disabled on session stop.
+* Client-side UI now logs location service events (entry/exit) events
+* Modified client-ui to include client-side logging capabilities.
+* Fixed an issue for Android API versions 28 or later where multiple WebViews cannot share the same data directory.
+* Added generic exception handling for exceptions that can occur when WebViews are accessed while OS is updating Chrome.
+
+The Project Griffon web UI now has new views specifically for users trying to inspect and improve Adobe Analytics and Location Service (Places) implementations.
+
+**Adobe Analytics View**
+The new Adobe Analytics view shows you events only related to your Adobe Analytics implementation. The list view now shows the action/state name and event “status” along with a newly formatted detail view. Status tells you when an SDK event is generated (processed), if the SDK has made a network request with Adobe Analytics (queued), and if we’ve returned post-processing information about that event (validated). This information should give you more confidence to determine if your context data is being mapped appropriately in Adobe Analytics.
+
+Generally speaking, for a given Analytics track event – the detailed view contains 3 valuable pieces:
+* Originating SDK Analytics request event
+* OOTB meta and context data from the request (such as report suite ID, SDK extension versions, OOTB context data, etc.) and
+* Post-processed information on the Analytics event (contains mapping of revars, evars, props, etc.).
+
+See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon) for documentation.
+
+**Location Service (Places) View**
+The new Location Services views allow you to inspect location entry and exit events on the Project Griffon web UI as well as an innovative on-device view. These handy views provide a convenient interface to view location specific data points for inspection on the web/client for in-context debugging depending on your business workflows.
```

Comment:
- Please add a line space between the heading and the text.
- Please reword this section as follows:
"The new Location Services views allow you to inspect the location entry and exit events on the Project Griffon web UI. This view includes an innovative on-device view, which provides a convenient interface to view location-specific data points for inspection on the web/client for in-context debugging depending on your business workflows.